src/components/profile/ProfileAccount.jsx

```diff
@@ -0,0 +1,142 @@
+import { useRequest } from "ahooks";
+import KeyIcon from "@mui/icons-material/Key";
+import PersonIcon from "@mui/icons-material/Person";
+import PhoneIcon from "@mui/icons-material/Phone";
+import { useForm } from "react-hook-form";
+import { useState } from "react";
+import { useAuth } from "@/models";
+
+export default function ProfileAccount() {
```

Comment:
I suggest writing profile page and edit page in two components.
For components that will not be reused, you could make a folder in pages called `profile` and put everything there

Regarding the edit, think of the below condition.

I would like to edit the profile, but I only want to change the phone, but i have to input all the thing.

For the form, please set the default value using the user's current profile

Reply:
The idea for the edit page is to retrieve and show all data if its already entered by the user earlier and if they wish to edit that they can rewrite and if not the same data is saved.